sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcQuerySuite.scala

```diff
@@ -742,6 +742,43 @@ abstract class OrcQuerySuite extends OrcQueryTest with SharedSparkSession {
       }
     }
   }
+
+  test("SPARK-36516: simple select queries with file meta cache") {
+    withSQLConf(SQLConf.FILE_META_CACHE_ORC_ENABLED.key -> "true") {
+      import org.scalatest.PrivateMethodTester._
+      import com.github.benmanes.caffeine.cache.stats.CacheStats
```

Comment:
Import only here, because only this case is using them

Reply:
For this one, shall we make a new test suite like `FileBasedDataSourceSuite` for both Parquet/ORC? For example, `FileMetaCacheSuite`?

Reply:
OK